cli/src/check_auth_status.rs

```diff
@@ -0,0 +1,24 @@
+use db::PgPool;
+use std::fs;
+use std::io::Read;
+use std::path::Path;
+use uuid::Uuid;
+
+pub async fn check_auth_status(pool: &PgPool) -> color_eyre::Result<()> {
+    let user_path = Path::new("auth.txt");
+    if user_path.exists() {
+        let mut file = fs::File::open("auth.txt")?;
+        let mut contents = String::new();
+        file.read_to_string(&mut contents)?;
+        let user_id = Uuid::parse_str(contents.trim())?;
+
+        match db::get_username_by_id(pool, user_id).await? {
+            Some(username) => println!("Logged in with Username: {username}, User ID: {user_id}"),
+            None => println!("file empty"),
+        }
+    } else {
+        return Err(color_eyre::eyre::eyre!("Not logged in."));
+    }
+    Ok(())
+
+}
```

Comment:
Did some refactoring here. I just moved it into a separate file. Let me know if you like this style or is there a specific style you would like me to do 🤔

Reply:
I like this!

I think instead of naming the module/file the same as the function maybe we can group by 'domain' a bit more?
So like maybe this is the `auth` module/file? and we can add more auth related stuff here as it exists?

No strong feels tho! Easy to move around later too lol